### DISCUSSION TITLE
Firefly III is unable to format monetary amounts because your server is missing the required packages.

### DISCUSSION THREAD

**Discussion by chboishabba** [firefly-iii/firefly-iii#9429] [Firefly III]
### Support guidelines

- [x] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [ ] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

locale -a
C
C.utf8
en_AU.utf8
en_NL.utf8
en_US.utf8
POSIX

Kernel: 6.11.5-arch1-1 

### Debug information

Debug information generated at 2024-11-05 02:16:01 for Firefly III version **v6.1.21**.

<table><thead><tr><th colspan="2">System information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Firefly III</td><td>6.1.21 / v2.1.0 / 24 (exp. 24)</td></tr><tr><td>PHP version</td><td>8.3.13 (64bits) / cli-server / Linux x86_64</td></tr><tr><td>BCscale</td><td>12</td></tr><tr><td>Error reporting</td><td>Display: Off, reporting: ALL errors</td></tr><tr><td>Max upload</td><td>2097152 (2 MB)</td></tr><tr><td>Database drivers</td><td><strong>*mysql*</strong>, </td></tr></tbody></table><table><thead><tr><th colspan="2">Firefly III information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Timezone</td><td>Europe/Amsterdam + Australia/Brisbane</td></tr><tr><td>App environment</td><td>production, debug: false</td></tr><tr><td>Layout</td><td>v1</td></tr><tr><td>Logging</td><td>notice, stack / (empty)</td></tr><tr><td>Cache driver</td><td>file</td></tr><tr><td>Default language and locale</td><td>en_US + equal</td></tr><tr><td>Trusted proxies</td><td></td></tr><tr><td>Login provider & user guard</td><td>eloquent / web</td></tr><tr><td>Login headers</td><td>N/A + N/A</td></tr><tr><td>Stateful domains</td><td></td></tr><tr><td>Last cron job</td><td>never (never)</td></tr><tr><td>Mailer</td><td>log</td></tr></tbody></table><table><thead><tr><th colspan="2">User-specific information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>User</td><td><span>#</span>1 of 1</td></tr><tr><td>User flags</td><td><span title="Has piggy banks">:pig:</span> <span title="Has rules">:wrench:</span> <span title="Has object groups">:bookmark_tabs:</span> <span title="Has subscriptions">:email:</span></td></tr><tr><td>Session start</td><td>2024-05-01 00:00:00</td></tr><tr><td>Session end</td><td>2024-11-06 23:59:59</td></tr><tr><td>View range</td><td>last365</td></tr><tr><td>User language</td><td>en_GB</td></tr><tr><td>User locale</td><td>en_GB</td></tr><tr><td>Locale(s) supported</td><td>en_GB.utf8: :x:<br>en_GB.UTF-8: :x:<br></td></tr><tr><td>User agent</td><td>Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36</td></tr></tbody></table>

**Comment by JC5** [comment]
Hi, thanks for opening an issue. 

You can also see that en_GB is not in the list that "locales -a" outputs, right?

Why do you think this is a bug?

**Comment by chboishabba** [comment]
Thank you for that fix.
It seems that GB locale does not appear on the help pages: (
https://docs.firefly-iii.org/how-to/firefly-iii/advanced/locales/)
It was also unclear which config.php one should check the locales of.
